src/software/ai/navigator/path_planner/hrvo/lv_agent.cpp

```diff
@@ -0,0 +1,62 @@
+#include "lv_agent.h"
+
+LVAgent::LVAgent(RobotId robot_id, const RobotState &robot_state, const RobotPath &path,
+                 double radius, double max_speed, double max_accel,
+                 double max_radius_inflation)
+    : Agent(robot_id, robot_state, path, radius, max_speed, max_accel,
+            max_radius_inflation)
+{
+}
+
+void LVAgent::computeNewVelocity(std::map<unsigned int, std::shared_ptr<Agent>> &robots,
+                                 Duration time_step)
+{
+    // TODO (#2496): Fix bug where LinearVelocityAgents go past their destination
+    // Preferring a velocity which points directly towards goal
+
```

Comment:
Lets close this, Im pretty sure it should be okay. The implementation seems sound and LV Agents don't have to be perfect given how fast we update their positions using the World